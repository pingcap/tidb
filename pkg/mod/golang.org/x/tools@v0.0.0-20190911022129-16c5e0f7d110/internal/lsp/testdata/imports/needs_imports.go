package imports //@import("package")

func goodbye() {
	fmt.Printf("HI")
	log.Printf("byeeeee")
}
