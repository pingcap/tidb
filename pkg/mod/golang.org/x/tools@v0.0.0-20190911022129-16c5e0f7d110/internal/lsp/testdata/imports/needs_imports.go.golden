-- goimports --
package imports //@import("package")

import (
	"fmt"
	"log"
)

func goodbye() {
	fmt.Printf("HI")
	log.Printf("byeeeee")
}

-- goimports-d --
--- imports/needs_imports.go.orig
+++ imports/needs_imports.go
@@ -1,5 +1,10 @@
 package imports //@import("package")
 
+import (
+	"fmt"
+	"log"
+)
+
 func goodbye() {
 	fmt.Printf("HI")
 	log.Printf("byeeeee")